packages/components/.storybook/preview.tsx

```diff
@@ -1,4 +1,5 @@
 import "@chanzuckerberg/czedi-kit-tokens/css/variables.css";
+import "../src/styles/fonts.css";
```

Comment:
_Why is this in `src/styles/` instead of `.storybook/`?_

In case we change our minds later and want to export it, or import it directly into `typography.tsx`, or something. no strong feeling tbh